Cargo.toml

```diff
@@ -308,425 +308,6 @@ altair-runtime = { path = "runtime/altair", default-features = false }
 centrifuge-runtime = { path = "runtime/centrifuge", default-features = false }
 
 # Build dependencies
-substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
-substrate-build-script-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.43" }
+substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
+substrate-build-script-utils = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
 vergen = "3.0.4"
-
-#
-# Cargo patch rules for all the paritytech/ based crates.
-#
-# With the rules below, we tell cargo that whenever it finds a crate with source in `paritytech/*`, it should use
-# the specific revision of the respective repository at hand, avoiding duplicated crates from tainting compilation.
-#
-[patch."https://github.com/paritytech/substrate"]
```

Comment:
I've removed all patch crates because most referenced repos from moonbeam have been removed. I guess we should deal with this again once we have most of the changes migrated and get errors with this.